README.md

```diff
@@ -115,159 +122,149 @@ _See documentation [here](#tensorflow-and-keras)._
 
 </details>
 
-> **3. Run the training like you are used to and start Aim UI**
-```shell
-$ aim up
-```
+**3. Run the training as usual and start Aim UI**
+
+_Prerequisite: In order to start Aim UI you need to have Docker installed._
 
-## Contents
-
-- [Aim](#aim)
-  - [Contents](#contents)
-  - [Getting Started In Three Steps](#getting-started-in-three-steps)
-  - [Installation](#installation)
-  - [Concepts](#concepts)
-  - [Where is the Data Stored](#where-is-the-data-stored)
-  - [Python Library](#python-library)
-    - [aim.track()](#track)
-    - [aim.set_params()](#set_params)
-    - [aim.Session()](#session)
-  - [Automatic Tracking](#automatic-tracking)
-    - [TensorFlow and Keras](#tensorflow-and-keras)
-    - [PyTorch Lightning](#pytorch-lightning)
-  - [Searching Experiments](#searching-experiments)
-    - [Search Examples](#search-examples)
-  - [Command Line Interface](#command-line-interface)
-    - [init](#init)
-    - [version](#version)
-    - [experiment](#experiment)
-    - [up](#up)
-    - [down](#down)
-    - [upgrade](#upgrade)
-    - [pull](#pull)
-  - [TensorBoard Experiments](#tensorboard-experiments)
-  - [Anonymized Telemetry](#anonymized-telemetry)
-  - [Contributor Guide](https://github.com/aimhubio/aim/wiki/Contributing)
-
-## Installation
-To install Aim, you need to have python3 and pip3 installed in your environment
-1. Install Aim python package
-```shell
-$ pip install aim
-```
-In order to start Aim UI you need to have Docker installed.
 ```shell
 $ aim up
 ```
+Jump to [[Overview](#overview)] [[SDK Specifications](#sdk-specifications)] [[Use Cases](#use-cases)]
 
-## Concepts
-Aim doesn't use too many concepts that needs explaining. Still, here are a few details to avoid confusion:
-- **Experiment**: it's a grouping mechanism for training runs. Experiments are a way to organize the training runs in a meaningful manner
-- **Run**: it's the single training run researchers start
+# Overview
 
-These two are separated just to enable effective organization of the research work.
+Aim helps you to compare 1000s of training runs at once via versatile SDK and flexible UI.
```

Comment:
I would avoid the adjectives "versatile" and "flexible". I think they don't add new info.
Maybe go with: `Aim helps you to compare 1000s of training runs at once through its framework-agnostic python SDK and performant UI`.
Alternatively, I would just stick to `Aim helps you to compare 1000s of training runs`

Reply:
Desc updated